README.md

```
@@ -38,6 +38,31 @@
 
+## Start learning in 30 seconds
+
+Your coding agent becomes your tutor. Two commands, no clone, no setup:
+
+```bash
+npx skills add rohitg00/ai-engineering-from-scratch
+```
+
+Then, inside your agent:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)